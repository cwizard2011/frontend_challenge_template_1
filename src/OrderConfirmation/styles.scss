.confirm-order-container {
  margin-top: 100px;
  margin-right: 100px;
  margin-left: 100px;
}

.h2-text-confirm {
  text-align: center;
  margin-bottom: 15px;
  font-family: 'Montserrat';
  font-weight: 600;
}

.hr-line {
  width: 50px;
  border-color: black;
}

.order-number-header {
  text-align: center;
  font-size: 27px;
  font-family: 'Montserrat';
  font-weight: 600;
  margin-bottom: 25px;
}

.order-details-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
}

.send-confirm {
  text-align: center;
  font-family: 'Montserrat';
  margin-bottom: 25px;
  color: #f7436b;
}

.order-header-row {
  font-weight: 700;
  font-family: 'Montserrat';
}

.order-values {
  opacity: .7;
  font-family: 'Montserrat';
}