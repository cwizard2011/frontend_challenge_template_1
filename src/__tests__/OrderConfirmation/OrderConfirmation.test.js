import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import OrderConfirmation from '../../OrderConfirmation';

describe('<OrderConfirmation />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<OrderConfirmation />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
