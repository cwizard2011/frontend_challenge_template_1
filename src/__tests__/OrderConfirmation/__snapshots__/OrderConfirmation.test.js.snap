// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderConfirmation /> renders without crashing 1`] = `<Spinner />`;
